cmd/mmsd/main.go

```diff
@@ -27,5 +30,25 @@ func main() {
 		WriteTimeout: 1 * time.Second,
 		IdleTimeout:  10 * time.Second,
 	}
+	startNATSServer()
+
+	log.Println("Starting webserver ...")
```

Comment:
Starting NATS server instead of webserver?

Reply:
There is a printout about starting the NATS server in the startNATSServer function.

Reply:
But I guess it would be easier to read if all web server stuff was in a startWebServer function and all nats stuff was in startNatsServer function.